docs/related-docs/avro-schema-document.mdx

```diff
@@ -0,0 +1,300 @@
+---
```

Comment:
Move to "Related Docs" section.
You should add Avro to the table in https://a.ml/docs/amf/amf_support and a link to this doc there. Change the name of "Constraints" to "Limitations" and do the same in the table that JSON Schema Document has (link to the specific section)

Reply:
Done